merky/test/exclude_annotation_nesting_test.py

```diff
@@ -0,0 +1,153 @@
+import collections
+
+from nose import tools
+from . import words
+import merky
+
+BASIC_SEQ = (words.SHEKELS, words.EUROS, words.EPEES)
+BASIC_SEQ_JSON = words.unify('[',
+                                '"', words.SHEKELS, '",',
+                                '"', words.EUROS, '",',
+                                '"', words.EPEES, '"',
+                             ']')
+BASIC_SEQ_HASH = '8129cb64b06e72be46eec5037eafc1583586325a'
+
+UNANNOTATED_NESTED_SEQ = (words.ANGSTROM, BASIC_SEQ, 213, -315)
+UNANNOTATED_NESTED_SEQ_JSON = words.unify('[',
+                                            '"', words.ANGSTROM, '",',
+                                            BASIC_SEQ_JSON, ',',
+                                            '213,',
+                                            '-315',
+                                          ']')
+UNANNOTATED_NESTED_SEQ_HASH = '14fe00c4e749eb96b8c66370902ac90a4f0d52fe'
+
+BASIC_DICT = {words.SHEKELS: words.EUROS, words.ANGSTROM: words.EPEES}
+BASIC_DICT_ORDERED = collections.OrderedDict((
+                        (words.ANGSTROM, words.EPEES),
+                        (words.SHEKELS, words.EUROS),
+                     ))
+BASIC_DICT_JSON = words.unify('{',
+                                '"', words.ANGSTROM, '":"', words.EPEES, '",',
+                                '"', words.SHEKELS, '":"', words.EUROS, '"',
+                              '}')
+BASIC_DICT_HASH = 'b22023acbbc3af979ba46d986706d8da7614f722'
+
+UNANNOTATED_NESTED_DICT = {"dict": dict(BASIC_DICT), "list": BASIC_SEQ}
+UNANNOTATED_NESTED_DICT_ORDERED = collections.OrderedDict((
+                                    ("dict", BASIC_DICT_ORDERED),
+                                    ("list", list(BASIC_SEQ))
+                                  ))
+UNANNOTATED_NESTED_DICT_JSON = words.unify('{',
+                                            '"dict":', BASIC_DICT_JSON, ',',
+                                            '"list":', BASIC_SEQ_JSON,
+                                           '}')
+UNANNOTATED_NESTED_DICT_HASH = '616bcc08fa64f4e8ed9cd7395c8981cdff045baf'
+UNANNOTATED_NESTED_DICT_TOKENIZED = collections.OrderedDict((
```

Comment:
Okay, I'm more confident about my issue above regarding the constant names.  In this case, you have the "UNANNOTATED_NESTED" prefix again, but that's actually how you're testing it below and how your "*_TOKENIZED" constant reflects things (expecting tokenization of the nested members).  So I think adjusting the prefix on the other guys will make things clearer.